deployment/README.md

```diff
@@ -22,4 +22,21 @@ By default `docker-compose` will pull Edna images from GitHub Container Registry
 However, if you import an image manually, it will be used instead.
 You can use `docker-compose pull` to update all images.
 
-Note: we use `.env` with default options, if you want to change it - create new file `.env.local` and run `docker-compose --env-file=.env.local start`
+Note: we use `.env` with default options, if you want to change it - create a new file (e. g. `.env.local`) and run `docker-compose --env-file=.env.local start`.
+
+## Configuration
+
+`edna-server` supports 3 ways of configuration: YAML configuration file, env variables and command line options.
+Here we use the first two:
+* [configuration file](./config.yaml) is mounted to `/config.yaml` and read by `edna-server` inside the image.
+* [.env](./.env) file defines a number of env variables propagated to the backend and other images:
+  + `POSTGRES_*` options affect both `postgres` and `backend` containers and should be self-descripting.
+  `backend` uses them to construct a connection string to connect to the DB.
+  + `EDNA_*` options match their counterparts from the configuration file.
+
+These are the most significant configuration options you may want to change:
+* `EDNA_API_LISTEN_ADDR` is the network address where the backend server will be bound.
```

Comment:
If somebody change it, there are can be a problem with frontend app which listens barcoded addr and port.. maybe we have to allow frontend app read some env variables to set listen addr and port

Reply:
I want to merge it before today's demo (i. e. ASAP), so I'll just remove it from this list for now.

Reply:
@gromakovsky okay
but create an issue please, in case it will be forgotten

Reply:
https://issues.serokell.io/issue/EDNA-146